Adding warning about potential duplicate builds (#9260)
circleci/circleci-docs: jekyll/_cci2/github-integration.adoc

[#installing-github-app-alongside-github-oauth-app]
### Installing GitHub App alongside the GitHub OAuth app

[CAUTION]
====

> Having both a GitHub App trigger *and* a GitHub OAuth trigger configured in the same project could result in duplicate builds. To prevent this behavior, make sure to disable the GitHub OAuth trigger. To do so:

Reviewer: ```suggestion
Having both a GitHub App trigger *and* a GitHub OAuth trigger configured in the same project could result in duplicate builds. To prevent this behavior, make sure to disable the GitHub OAuth trigger. To do so, follow these steps:
```

> . Navigate to the GitHub repository associated with the project.
> . Go to "Settings", then "Webhooks".
> . Locate the CircleCI webhook (`https://circleci.com/hooks/github`) and click "Delete".

Reviewer: ```suggestion
. Navigate to the GitHub repository associated with the project.
. Go to menu:Settings[Webhooks].
. Locate the CircleCI webhook (`https://circleci.com/hooks/github`) and select btn:[Delete].
```

> ====

Reviewer 2: @yaningo  Could we add something like this below please?

> Note: Deleting the OAuth webhook will prevent environment variable CIRCLE_PULL_REQUEST from being populated in all pipelines. We are working to address this issue.

[#installing-github-app-alongside-github-oauth-app]
### Installing GitHub App alongside the GitHub OAuth app

[CAUTION]
====
Having both a GitHub App trigger *and* a GitHub OAuth trigger configured in the same project could result in duplicate builds. To prevent this behavior, make sure to disable the GitHub OAuth trigger. To do so, follow these steps:

. Navigate to the GitHub repository associated with the project.
. Go to menu:Settings[Webhooks].
. Locate the CircleCI webhook (`https://circleci.com/hooks/github`) and select btn:[Delete].

> _Note: Deleting the OAuth webhook will prevent the xref:variables/#built-in-environment-variables[built-in environment variable] `CIRCLE_PULL_REQUEST` from being populated in all pipelines._

Reviewer: ```suggestion
_Note: Deleting the OAuth webhook will prevent the xref:variables#built-in-environment-variables[built-in environment variable] `CIRCLE_PULL_REQUEST` from being populated in all pipelines._
```
